components/Footer/src/index.scss

```diff
@@ -0,0 +1,95 @@
+.denhaag-footer {
+  --utrecht-paragraph-color: #fff;
+  --utrecht-paragraph-font-size: 16px;
+  --utrecht-paragraph-line-height: normal;
+  --utrecht-paragraph-margin-block-start: 12px;
+  --utrecht-paragraph-margin-block-end: 12px;
+
+  background-color: #4b4b4b;
+
+  --utrecht-link-social-background-color: transparent;
```

Comment:
And these tokens could be set in the Proprietary folder too? Or do we use the `link-social` compponent on other (light) backgrounds too?